exercises/yacht/test/Tests.hs

```diff
@@ -0,0 +1,167 @@
+{-# LANGUAGE RecordWildCards #-}
+
+import Data.Foldable     (for_)
+import Test.Hspec        (Spec, describe, it, shouldBe)
+import Test.Hspec.Runner (configFastFail, defaultConfig, hspecWith)
+
+import Yacht (yacht, Category(..))
+
+main :: IO ()
+main = hspecWith defaultConfig {configFastFail = True} specs
+
+specs :: Spec
+specs = describe "yacht" $ for_ cases test
+  where
+
```

Comment:
```suggestion
```

Reply:
Cool. I thought it seemed odd to have the space here, but I saw it in another test, so I went with the space ;) Fixed in this upcoming round of commits :smile: